packages/shorebird_cli/lib/src/commands/patch/ios_patcher.dart

```diff
@@ -312,17 +314,24 @@ class IosPatcher extends Patcher {
 
     final linkProgress = logger.progress('Linking AOT files');
     double? linkPercentage;
-    try {
-      final dumpDebugInfoDir = await aotTools.isLinkDebugInfoSupported()
-          ? Directory.systemTemp.createTempSync()
-          : null;
+    final dumpDebugInfoDir = await aotTools.isLinkDebugInfoSupported()
+        ? Directory.systemTemp.createTempSync()
+        : null;
 
-      if (dumpDebugInfoDir != null) {
-        logger.detail(
-          'Dumping link debug info to ${dumpDebugInfoDir.path}',
-        );
+    Future<void> consolidateDebugInfo({bool warn = false}) async {
+      if (dumpDebugInfoDir == null) return;
+
+      final debugInfoZip = await dumpDebugInfoDir.zipToTempFile();
+      debugInfoZip.copySync(p.join('build', debugInfoFile.path));
+      final msg = 'Link debug info saved to ${debugInfoFile.path}';
```

Comment:
I think it's fine to just always log this as a detail since we always put the debug info in the same relative location.

Reply:
Just wanted to make it more explicit to the user that the debug data exists, so it would probably save us support time explaining where those files are created.

But I can change it to detail, no strong opinion